File: java/04_loops.java
public class Loops {
    public static void main(String[] args) {
        for (int i = 0; i < 5; i++) {
            System.out.println(i);
        }

        String[] fruits = {"apple", "banana", "cherry"};
        for (String fruit : fruits) {
            System.out.println(fruit);
        }

        int count = 0;
        while (count < 3) {
            System.out.println("count is " + count);
            count++;
        }
    }
}

File: rust/04_loops.rs
fn main() {
    for i in 0..5 {
        println!("{}", i);
    }

    let fruits = ["apple", "banana", "cherry"];
    for fruit in fruits.iter() {
        println!("{}", fruit);
    }

    let mut count = 0;
    while count < 3 {
        println!("count is {}", count);
        count += 1;
    }
}
